### ISSUE TITLE
Is it possible to use this plugin without a Resource Pool?

### ISSUE THREAD

**Issue by viktorbenei** [nsidc/vagrant-vsphere#166]
So far we did not create a Resource Pool in our vSphere, I was just wondering if there's a way to use this plugin without a Resource Pool, or we really have to create one.

**Comment by edmcman** [comment]
I use it without a resource pool.

**Comment by viktorbenei** [comment]
Maybe this is just an issue with the latest version of the plugin - we used the previous one with another vSphere network, and for the new one I upgraded to the latest plugin version - but I always get:

```
vagrant up --provider=vsphere
Bringing machine 'default' up with 'vsphere' provider...
==> default: Calling vSphere CloneVM with the following settings:
==> default:  -- Source VM: ...
==> default:  -- Target VM: ...
InvalidArgument: A specified parameter was not correct: spec.location.pool
```

Just created a Resource Pool, and doesn't really matter whether I set the pool or not, the error is always the same.

**Comment by viktorbenei** [comment]
The `Vagrantfile` is the same as the one we used for the other vSphere network (where it works perfectly with the previous version of this plugin), just changed the values (host, psw, `compute_resource_name`, etc) to match the new vSphere network configuration.

**Comment by viktorbenei** [comment]
Just tried with `v1.4.1` and got the same error..

**Comment by edmcman** [comment]
I have both compute_resource_name and resource_pool_name unset.

**Comment by viktorbenei** [comment]
I think I found the issue, at least it works now. I removed every other (non login) parameter except:

* `compute_resource_name`
* `template_name`
* `name`

It works if only there parameters are defined, but once I set `clone_from_vm = true` it fails with the error `InvalidArgument: A specified parameter was not correct: spec.location.pool`.

The error is actually that while we used VM with linked clone on our other vSphere network we used templates this time, so `clone_from_vm` should be false. This is of course a PEBKAC type of error, but if possible the error message could be a bit more specific 😉
